pkg/expr/mathexp/funcs.go

```diff
@@ -29,6 +29,21 @@ var builtins = map[string]parse.Func{
 		Return: parse.TypeScalar,
 		F:      null,
 	},
+	"round": {
+		Args:          []parse.ReturnType{parse.TypeVariantSet},
+		VariantReturn: true,
+		F:             round,
+	},
+	"ciel": {
```

Comment:
I think this is a spelling mistake right?

```suggestion
	"ceil": {
```